nstar/src/commands.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2019 - 2020 ESRLabs
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+use clap::{AppSettings, Result};
+use npk::manifest::Version;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+pub enum Northstar {
+    #[structopt(about = "List available containers", alias = "ls", alias = "list")]
+    Containers,
+    #[structopt(about = "List configured repositories", alias = "repos")]
+    Repositories,
+    #[structopt(about = "Start a container")]
+    Start {
+        #[structopt(help = "Name of the container")]
+        name: Option<String>,
+    },
+    #[structopt(about = "Stop a container")]
+    Stop {
+        #[structopt(help = "Name of the container")]
+        name: Option<String>,
```

Comment:
Instead of `Option<Stirng>` use an `String` and add a boolean flag `-a,-all`. Same could be good for `start`.